otherlibs/stdune/src/stdune.ml

```diff
@@ -1,5 +1,4 @@
-[@@@alert unstable "The API of this library is not stable and may change without notice."]
-[@@@alert "-unstable"]
+
```

Comment:
an accident?

Reply:
It was the quickest hack to make my build pass with OCaml 5.2, I pushed it by mistake.